src/tools/translate-to-fuzz.h

```diff
@@ -639,9 +639,14 @@ class TranslateToFuzzReader {
           }
         }
         switch (conditions) {
-          case 0: if (!oneIn(4)) continue;
-          case 1: if (!oneIn(2)) continue;
-          default: if (oneIn(conditions + 1)) continue;
+          case 0:
```

Comment:
since the body isn't on the same line, please add `{,}` for each (see other switches for the convention)

Reply:
Fixed in 90364e3.